src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ExportProvider.cs

```diff
@@ -138,11 +140,17 @@ private byte[] ExportPfx(SafePasswordHandle password)
                         throw Interop.Crypto.CreateOpenSslCryptographicException();
                     }
 
-                    return Interop.Crypto.OpenSslEncode(
+                    var result = Interop.Crypto.OpenSslEncode(
                         Interop.Crypto.GetPkcs12DerSize,
                         Interop.Crypto.EncodePkcs12,
                         pkcs12);
+
+                    // ensure certs handle isn't finalized while raw handle(s) is in use
```

Comment:
Grammar: Something like "ensure cert handles aren't finalized while the raw handles are in use"